code/utils.py

```diff
@@ -615,6 +615,18 @@ def write_c1_grad_script(origsFile):
 
 # *********************************************************************************************************************
 
+def time_in_range(start, end, x):
```

Comment:
check the function check_times in the class variables.py 
Is there some refactoring that could have been done here to leverage this instead of creating a new function with the same functionality. @cgoodale